packages/vite/src/node/ssr/ssrModuleLoader.ts

```diff
@@ -203,17 +204,21 @@ async function instantiateModule(
     )
   } catch (e) {
     mod.ssrError = e
+
     if (e.stack && fixStacktrace) {
       ssrFixStacktrace(e, moduleGraph)
-      server.config.logger.error(
-        `Error when evaluating SSR module ${url}:\n${e.stack}`,
-        {
-          timestamp: true,
-          clear: server.config.clearScreen,
-          error: e,
-        },
-      )
     }
+
+    server.config.logger.error(
+      `Error when evaluating SSR module ${url}:` +
+        (e.importee ? ` failed to import '${e.importee}'\n` : '\n'),
+      {
+        timestamp: true,
+        clear: server.config.clearScreen,
+      },
+    )
```

Comment:
Tiny nit: For the importee quotes, I think we should keep it double-quotes for consistency with the other messages in the codebase. And also keep the `error: e` as it's used to track if an error is already logged by other tools in the ecosystem.

```suggestion
    server.config.logger.error(
      `Error when evaluating SSR module ${url}:` +
        (e.importee ? ` failed to import "${e.importee}"\n` : '\n'),
      {
        timestamp: true,
        clear: server.config.clearScreen,
        error: e
      },
    )
```

I also think always logging an error here is a good move, not sure why we didn't before, but this should make the error clearer now.